app/js/controllers/settings.js

```diff
@@ -5,21 +5,39 @@
 export default class SettingsController extends Controller {
   constructor(options) {
     super(options);
+
+    this.merging = false;
+
+    this._refreshAddons = this._refreshAddons.bind(this);
   }
 
   open() {
     this.view.modal.open();
 
-    AddonService.getAddons(window.location.host).then((addons) => {
-      this.view.setAddons(addons);
+    this._refreshAddons();
+
+    ['install', 'uninstall', 'enabledstatechange'].forEach((e) => {
```

Comment:
After reading this function, it looks like it only checks for add-ons that target the current app. The user could install add-ons some other way, which target this specific app. We want to only display Customizer-generated add-ons in this list. We discussed this briefly a while ago. One suggestion I had was to add something to the generated manifest, like ```"customizer": "true"```, or some other meta-data.